napbot/extensions/music/song.py

```diff
@@ -73,7 +98,8 @@ def title_slugify(title: str) -> str:
 
 
 class Song:
-    def __init__(self, audio_path: str, log: Logger):
+    song_count = 0
```

Comment:
unused variable

Reply:
static variable, it keeps track of number of songs from before